server/src/controllers/sessions.controllers.ts

```diff
@@ -0,0 +1,52 @@
+import { AnyAaaaRecord } from "dns";
+import Session from "../models/SessionModel";
+import { Request, Response } from 'express';
+
+const createSession = async (req: Request, res: Response) => {
+    try {
+        const {
+            sessionDate,
+            region,
+            staff,
+            archived,
+            attendance
+        } = req.body;
+
+        const newSession = await Session.create(req.body);
+        await newSession.save();
+
+        return res.status(200).json(newSession);
+    } catch (error: any) {
+        console.error(error.message);
```

Comment:
Please handle errors in typesafe manner. See example here: https://github.com/ChangePlusPlusVandy/vac-net/blob/a9c4e9bab9f8ff75490e1b4e561bcb6290a39fbc/server/src/controllers/session.controllors.ts#L19